cmd/cerbos/repl/internal/repl.go

```diff
@@ -534,6 +541,48 @@ func (r *REPL) mkEnv() (*cel.Env, error) {
 	return conditions.StdEnv.Extend(cel.Declarations(decls...))
 }
 
+func isTerminated(line string, stack *runeStack) bool {
+	if line[len(line)-1] == '\\' {
```

Comment:
We need to remove the `\` from the end of the string as well. Otherwise, the input is invalid.

```
-> :let request = {\
>   "principal":{\
>     "id":"john",\
>     "roles":["employee"],\
>     "attr":{"scope":"foo.bar.baz.qux"}\
>   },\
>   "resource":{\
>     "id":"x1",\
>     "kind":"leave_request",\
>     "attr":{"scope":"foo.bar"}\
>   }\
> }
Error: Failed to process input
 failed to unmarhsal JSON as "request": proto: syntax error (line 1:3): invalid value \
```